src/lm_buddy/cli/run.py

```diff
@@ -1,6 +1,13 @@
 import click
 
-from lm_buddy.integrations.wandb import WandbArtifactLoader
+import lm_buddy
```

Comment:
could we import run_job directly here instead of the package?

Reply:
We could, but I somewhat like the syntax of calling `lm_buddy.run_job`. It feels natural as the sole function of the package.